governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Add LINK-A to Liquidations 2.0 Framework - April 5, 2021
+summary: This poll proposes system parameters which could be used to initialize LINK-A with the Liquidations 2.0 Framework.
```

Comment:
Change to: "This poll proposes system parameters to be used in initializing LINK-A with the Liquidations 2.0 Framework"